frontend/public/components/machine-set.tsx

```diff
@@ -299,10 +353,27 @@ export const MachineSetList: React.SFC = (props) => {
               obj.metadata.namespace,
             )}/machines`}
           >
-            {getReadyReplicas(obj)} {t('public~of')} {getDesiredReplicas(obj)}{' '}
-            {t('public~machines')}
+            {t('public~{{numReplicas}} of {{numDesiredReplicas}} machine', {
+              numReplicas: getReadyReplicas(obj),
+              numDesiredReplicas: getDesiredReplicas(obj),
+              count: getDesiredReplicas(obj),
+            })}
           </Link>
         </TableData>
+        <TableData>{getMachineSetInstanceType(obj) || '-'}</TableData>
+        <TableData className={tableColumnClasses[2]}>
+          {relatedNodes.reduce((a, b) => a + parseInt(b.status.capacity.cpu, 10), 0)}
+        </TableData>
+        <TableData className={tableColumnClasses[2]}>
+          {t('public~{{memory}} GiB', {
+            memory: Math.round(
+              relatedNodes.reduce(
+                (a, b) => a + parseInt(b.status.capacity.memory.replace('Ki', ''), 10),
```

Comment:
Same comments as above. Don't assume that memory will be specified in `Ki`. Use the helpers in units to get the value in bytes and to format the result.